views/default/river/elements/responses.php

```diff
@@ -33,7 +33,7 @@
 		'subtype' => 'comment',
 		'container_guid' => $object->getGUID(),
 		'limit' => 3,
-		'order_by' => 'e.time_created desc',
+		'order_by' => [new OrderByClause('time_created', 'DESC')],
```

Comment:
Missing a use statement, so this will probably crash

Reply:
fixed

Reply:
Also missing in the first file, I think

Reply:
no, was already present there